### PR TITLE
🌱 Validate OWNERS files - remove sections about emeritus reviewers.

### PR DESCRIPTION
**Pull request by peizhouyu**
please see [kubernetes/community#6334](https://github.com/kubernetes/community/issues/6334)

What type of PR is this?   

/kind cleanup

What this PR does / why we need it:

Which issue(s) this PR fixes: 

Fixes #

Special notes for your reviewer:

Does this PR introduce a user-facing change?  
```
NONE
```
Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:
```

```

### CODE REVIEW

**Review by fabriziopandini — COMMENTED** [review]
/lgtm

**Comment by killianmuldoon** [comment]
/lgtm

Just wondering why the signoff in the PR description is:
> Signed-off-by: Davanum Srinivas davanum@gmail.com

I don't think it matters - is it just copy-paste?

**Comment by sbueringer** [comment]
/lgtm
 
> Just wondering why the signoff in the PR description is:
> 
> > Signed-off-by: Davanum Srinivas [davanum@gmail.com](mailto:davanum@gmail.com)
> 
> I don't think it matters - is it just copy-paste?

@peizhouyu Can you please remove it from the PR description? Thank you :)

**Comment by peizhouyu** [comment]
> /lgtm
> 
> > Just wondering why the signoff in the PR description is:
> > > Signed-off-by: Davanum Srinivas [davanum@gmail.com](mailto:davanum@gmail.com)
> > 
> > 
> > I don't think it matters - is it just copy-paste?
> 
> @peizhouyu Can you please remove it from the PR description? Thank you :)
Thanks for spotting the problem, I't was removed.
Thank you

File: OWNERS [hunk 1]
```diff
@@ -16,8 +16,4 @@ emeritus_approvers:
   - kris-nova
   - ncdc
   - roberthbailey
-  - davidewatson
-
-emeritus_maintainers:
-  - detiber
-  - ncdc
+  - davidewatson
```
Comment (mkumatag) [inline OWNERS:19]: add a newline here in the end

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: OWNERS (hunk 1 above)
Comment (peizhouyu) [reply OWNERS]: > add a newline here in the end
I've already added it，Thanks.

**Review by mkumatag — COMMENTED** [review]
/lgtm

**Review by fabriziopandini — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**